### ISSUE TITLE
Disabling panning should disable panning on touch devices

### ISSUE THREAD

**Issue by geroyche** [highcharts/highcharts#17445]
#### Expected behaviour
Setting panning: false disables all panning

#### Actual behaviour
On desktop, using pankey, nothing happens - as expected.
But I can still pan, e.g. when zoomed in, on mobile (may require two fingers).

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/chart/pankey/
it's the actual highcharts demo for panning... just disable panning there.

#### Product version
Highcharts 10.1.0

**Comment by pawellysy** [comment]
Hi @geroyche, thanks for creating this ticket. 
Generally, some behaviors are different between mobile and computer browsers. The panning is an essential part of mobile charts, and when zooming, using 2 fingers to zoom and pan a chart is a pretty natural behavior in my opinion. Could you describe more thoroughly, how would you expect the chart zooming and panning to work on mobile? Thanks a lot!

**Comment by geroyche** [comment]
ok, there is a backstory...

I'm loading monthly data from a server, and with panning enabled users might pan to the edges of the month and then hit a wall.
I wish there were events that indicated when a panning event reaches either edge of data, so that I could then load more data from the server. I realize there are some solutions on s/o and elsewhere, but none that really worked for me. I tapped into afterSetExtremes, which seems to the place, but ran into some issues that tie into my outer logics.

Anyway, one of the underlying issues with panning for me is that when you pan in a diagram and reach the edge of the data range, further panning will suddenly start to zoom in on the data range, as opposed to maybe bounce but keep the current zoom level. my client found this confusing. ultimately we've come to the decision to generally disable touch panning. I added some ui elements instead (buttons) that call setExtremes on click (and load data from the server if min or max indicated the requested timeframe is in a different month). this solution is satisfactory, were it not for the fact that I cannot, as promised, disabled the built-in panning.

to the broader point: in my opinion it's not about whether panning feels natural or not. a boolean option that promises to disable panning should disable it consistently, no?

so to sum up: since I allow zoom and pinch, using two finger to pinch zoom should work, as it does indeed. since I disallow panning, trying to pan the diagram with one finger (followTouchMove: false) or two fingers (followTouchMove: true) should not result in any visible movement of the diagram. this is what "panning: false" does for mouse interactions, and it should be a consistent experience from a developers perspective in my view.

**Comment by pawellysy** [comment]
Thanks @geroyche  for a complete back story, It does make sense. I will mark this ticket as an enhancement and we will look at this soon. Thanks again!
